components/splitbutton/overview.md

```diff
@@ -0,0 +1,144 @@
+---
+title: Overview
+page_title: SplitButton Overview
+description: Overview of the SplitButton for Blazor.
+slug: splitbutton-overview
+tags: telerik,blazor,splitbutton
+published: True
+position: 0
+---
+
+# Blazor SplitButton Overview
+
+The <a href = "https://www.telerik.com/blazor-ui/splitbutton" target="_blank">SplitButton for Blazor</a> is a combination of a button and a dropdown. It provides a collection of related user actions in compact interface. The SplitButton has one primary clickable action, which is always visible, and a list of secondary actions that are displayed in a dropdown when the user clicks on the arrow.
+
+## Creating Blazor SplitButton
+
+1. Add a `<TelerikSplitButton>` tag.
+1. Define the primary action in a child `<SplitButtonContent>` tag. Its content can be plain text, HTML or even another component.
+1. Set the `OnClick` parameter of the `<TelerikSplitButton>` tag. This will be the event handler for the primary action.
+1. Add a child `<SplitButtonItems>` tag. Insert some `<SplitButtonItem>` tags inside it.
```

Comment:
We have explained these as instances of the `<X>` tag, at least this far.

I rather like an explanation like:

"Insert some `instances` of the `<SplitButtonItem>` tag inside it"

This is not something mandatory, just wanted to mention it.

Reply:
I appreciate the consistency, but isn't "instances of <x>" long and unnecessary? The main question should be "what is the simplest and shortest explanation that is technically correct and doesn't cause confusion". Let me know what you think.

Reply:
You are correct, we might stick with this shorter version in future docs.